### ISSUE TITLE
Should the current location on a breadcrumb trail be an `<a>` anchor element?

### ISSUE THREAD

**Issue by verdansk** [w3c/aria-practices#3047]
### Issue

There are two accessibility documents conflicting on the topic of implementing breadcrumbs. From these docs it's unclear on how the last item in the breadcrumb list should be structured. 

<hr>

### Example 1

In [this example](https://www.w3.org/WAI/ARIA/apg/patterns/breadcrumb/examples/breadcrumb/#sc1_label) the last item is structured as an `<a>` anchor-element together with the `aria-current`-attirbute:

```
// Example from: https://www.w3.org/WAI/ARIA/apg/patterns/breadcrumb/examples/breadcrumb/
<nav aria-label="Breadcrumb" class="breadcrumb">
  <ol>
    <li>
      <a href="../../../..">
        WAI-ARIA Authoring Practices Guide (APG)
      </a>
    </li>
    <li>
      <a href="../../../">
        Patterns
      </a>
    </li>
    <li>
      <a href="../../">
        Breadcrumb Pattern
      </a>
    </li>
    <li>
      <a href="" aria-current="page">
        Breadcrumb Example
      </a>
    </li>
  </ol>
</nav>
```

<hr>

### Example2

However [in this document ](https://www.w3.org/TR/WCAG20-TECHS/G65.html) the last item on the breadcrumb trail should not be an `<a>` anchor element, as we can digest from the test-scenarios (5a, 5b). But also from [the third example](https://www.w3.org/TR/WCAG20-TECHS/G65.html#G65-examples).

> Procedure
> When breadcrumb trails have been implemented in a set of Web pages:
> 1. Navigate to a Web page.
> 2. Check that a breadcrumb trail is displayed.
> 3. Check that the breadcrumb trail displays the correct navigational sequence to reach the current location or the correct hierarchical path to the current location within the site structure.
> 4. For a breadcrumb trail that does not include the current location:
> a. Check that all elements in the breadcrumb trail are implemented as links.
> 5. For a breadcrumb trail that does include the current location:
> a. Check that all elements except for the current location are implemented as links.
> b. Check that the current location is not implemented as a link.
> 6. Check that all links navigate to the correct Web page as specified by the breadcrumb trail.

<hr>

### Question

So this leaves me with the question what the accessibility best practices should be when it comes to the last item on a breadcrumb trail? From a UX-perspective I can see why it should be an `<a>` anchor-element, so that it can be focusable, but on the other hand it doesn't make sense to have an empty `<a>` anchor-element or even a link to the current page in the breadcrumb trail. As I don't see the added value of having those.

Anybody who can shed some light on this?

**Comment by mcking65** [comment]
Per discussion in July 2, 2024 meeting,raised [Modify G65 to support implementing current location in a breadcrumb trail as a link with \`aria-current="page"\` · Issue #3948 · w3c/wcag](https://github.com/w3c/wcag/issues/3948)

**Comment by mcking65** [comment]
@verdansk 

Thank you for the question. If the meeting minutes and issue I raised regarding WCAG technique G65 do not provide a sufficient answer, please feel free to re-open this issue.

**Comment by mcking65** [comment]
Waiting for publication of this merged pull request:
[Update G65.html by scottaohara · Pull Request #3951 · w3c/wcag](https://github.com/w3c/wcag/pull/3951)
